style.css

```diff
@@ -563,7 +536,451 @@ ul{
 .icon-btn{
     margin-left: 5px
 }
+.desktop{
+    display: none;
+}
+/* Desktop Verion Styling */
+@media (min-width: 768px) {
+    html, body {
+        margin: 0;
+        height: 100%;
+        width: 100%;
+        max-width: 100%;
+        min-width: 100%;
+    }
+    body {
+        box-sizing: border-box;
+    }
+    .desktop{
+        display: contents;
```

Comment:
You could change this to `display: none;` to stop the nav items mentioned above from displaying on top of each other.

Reply:
Thank you for the suggested, well noted.